splinterd/tests/framework/network.rs

```diff
@@ -20,16 +20,21 @@ use cylinder::{secp256k1::Secp256k1Context, Context};
 use splinter::error::{InternalError, InvalidArgumentError};
 use splinter::registry::Node as RegistryNode;
 use splinter::threading::lifecycle::ShutdownHandle;
-use splinterd::node::{Node, NodeBuilder, RestApiVariant, ScabbardConfigBuilder};
+use splinterd::node::{Node, NodeBuilder, RestApiVariant, RunnableNode, ScabbardConfigBuilder};
 use tempdir::TempDir;
 
 pub struct Network {
     default_rest_api_variant: RestApiVariant,
-    nodes: Vec<Node>,
+    nodes: Vec<NetworkNode>,
     temp_dirs: HashMap<String, TempDir>,
     external_registries: Option<Vec<String>>,
 }
 
+pub enum NetworkNode {
+    RunningNode(Node),
```

Comment:
This could be simplified to Node(Node), RunnableNode(RunnableNode).

Reply:
Updated